BD-2306: Adding examples to teams doc (#5930)
braze-inc/braze-docs: _docs/_user_guide/administrative/app_settings/manage_your_braze_users/teams.md

### Example

Consider the following two scenarios for a marketer in Braze named Michelle. Michelle is a member of a team called "Development". She has access to all of the team-level permissions for the Development team.

{% tabs %}
{% tab Scenario 1: Only team permissions %}

In this scenario, Michelle is a limited user that has no workspace-level permissions. Her permissions look something like this:

![]({% image_buster /assets/img_archive/scenario1.png %})

Based on Michelle's assigned permissions, whenever she creates a campaign, she can only assign the "Development" team to that campaign. She can't launch the campaign unless the team is assigned, and she can't view or access any other team tags.

![]({% image_buster /assets/img_archive/team_permissions_scenario1.gif %})

{% endtab %}
{% tab Scenario 2: Team permissions and workspace permissions %}

In this scenario, Michelle is still a member of the Development team, but she also has an additional workspace-level permission.

![]({% image_buster /assets/img_archive/scenario2.png %})

> Because Michelle has the workspace-level permission of "Access Campaigns, Canvases, Cards, Content Blocks, Feature Flags, Segments, Media Library, and Preference Centers", she can view and assign other team filters to the campaign she creates.

Author: ```suggestion
Because Michelle has the workspace-level permission of "Access Campaigns, Canvases, Cards, Content Blocks, Feature Flags, Segments, Media Library, and Preference Centers", she can view and assign other team filters to the campaign she creates.
```

![]({% image_buster /assets/img_archive/team_permissions_scenario2.gif %})

> Similar to the first scenario, Michelle must add a team to the campaign before she can launch it.

Author: ```suggestion
Similar to the first scenario, Michelle must add the Development team tag to the campaign before she can launch it.
```

To do so, create a "Development" team that only has access to test users. You can limit a team to only access test users if your test users are identifiable by a custom attribute. Then, add the custom attribute as a definition when creating or editing the team (see the preceeding section [Creating teams](#creating-teams)). Your approvers should have access to all users.

The general process would be as follows:

1. Development team creates a campaign and adds the "Development" team tag.
2. Development team launches the campaign to test users.
3. Approver team validates the local campaign design, promotes, and launches. To launch, the Approver team changes the team tag from "Development" to "[All Teams]" and relaunches the campaign.

For changes to active campaigns:

1. Development team clones the running campaign, adds the "Development" team tag, and saves.
2. Development team makes edits and shares with the Approver team.
3. Approver team removes the "Development" team tag, pauses the previous campaign, and launches the new campaign.

## Archiving an existing Team

> You can archive Teams from the **Internal Teams** page.

Author: ```suggestion
You can archive teams from the **Internal Teams** page.
```

> Select one or many teams to archive. If the Team is not associated with any object within Braze, the Team will be archived immediately. If the Team is associated with an object, you will be presented with an option to remove the Team after the archive process or replace the Team.

Author: ```suggestion
Select one or many teams to archive. If the team is not associated with any object within Braze, the team will be archived immediately. If the team is associated with an object, you will be presented with an option to remove the team after the archive process or replace the team.
```
